### PR TITLE
Fixed the periodic_task definition

### PR DESCRIPTION
**Pull request by cobusc**
The periodic task now iterates over all defined feeds.

### CODE REVIEW

File: rssfeed/tasks.py [hunk 1]
```diff
@@ -41,6 +35,15 @@ def poll_feed(pk_feed, verbose=False):
     parse_entries(parsed, db_feed, verbose)
 
 
+@periodic_task(run_every=crontab(minute="*/15"), ignore_result=True)
+def poll_feeds(verbose=False):
```
Comment (chrislombaard) [inline rssfeed/tasks.py:39]: Will this still work when a new feed is added in `models.py`? line 33

Comment (chrislombaard) [reply rssfeed/tasks.py:39]: Oh I see its called `polled_feeds`

Comment (chrislombaard) [reply rssfeed/tasks.py:39]: I'm happy with that.

Comment (cobusc) [reply rssfeed/tasks.py:39]: Yeah, `poll_feeds()` calls `poll_feed()` for every feed defined.

**Review by chrislombaard — APPROVED** [review]
+1

**Pull request merged**